__tests__/app.test.js

```diff
@@ -82,3 +82,49 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 sends an array of comment objects for the specified article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toEqual(expect.any(Array));
+        expect(comments.length).toBe(11);
+        comments.forEach((comment) => {
+          expect(Object.keys(comment).length).toEqual(6);
+          expect(comment.comment_id).toEqual(expect.any(Number));
+          expect(comment.votes).toEqual(expect.any(Number));
+          expect(comment.created_at).toEqual(expect.any(String));
+          expect(comment.author).toEqual(expect.any(String));
+          expect(comment.body).toEqual(expect.any(String));
+          expect(comment.article_id).toEqual(expect.any(Number));
```

Comment:
since we're requesting comments from an article with a specific ID - this is the one property you can directly assert